### PR TITLE
[TODO] Create issues for the TODO comments

### PR DESCRIPTION
**Pull request by 4TT1L4**
Related to:
 - #18

### CODE REVIEW

**Comment by 4TT1L4** [comment]
I made a mistake and inserted some tabs:
```
Tab character found here, and in two further locations.
```
I need to go through the changes and replace these with spaces.

File: src/GeniusYield/Types/Value.hs [hunk 1]
```diff
@@ -598,9 +598,10 @@ parseAssetClassCore sep tkParser t = Atto.parseOnly parser (TE.encodeUtf8 t)
 
 -- | Token name is an arbitrary byte string up to 32 bytes long.
 --
--- TODO: it's unclear whether it's an arbitrary byte string
--- or UTF8 encoded text (which encoded byte form is 32 byte long at most).
+-- TODO: it's unclear whether it's an arbitrary byte string or UTF8 encoded text #32
+-- (which encoded byte form is 32 byte long at most).
 -- /We treat it as an arbitrary string/.
+-- (https://github.com/geniusyield/atlas/issues/32)
```
Comment (sourabhxyz) [inline src/GeniusYield/Types/Value.hs:604]: could you move this above the "We treat it ..." line

File: src/GeniusYield/CardanoApi/EraHistory.hs [hunk 2]
```diff
@@ -19,7 +19,7 @@ import qualified Ouroboros.Consensus.Util.Counting as Ouroboros
 
 This is safe as long as Ouroboros.Interpeter is defined as a newtype to Ouroboros.Summary.
 
-TODO: Is there a way to ensure, at runtime, that Interpreter is still a newtype to Summary?
+TODO: Can we ensure runtime, that Interpreter is still a newtype to Summary? #20 (https://github.com/geniusyield/atlas/issues/20)
```
Comment (sourabhxyz) [inline src/GeniusYield/CardanoApi/EraHistory.hs:22]: _Can we ensure **at** runtime_, rest is fine.

**Review by sourabhxyz — CHANGES_REQUESTED** [review]

File: src/GeniusYield/CardanoApi/EraHistory.hs (hunk 2 above)
Comment (4TT1L4) [reply src/GeniusYield/CardanoApi/EraHistory.hs]: Fixed in 26b30b545432f3d6535bf48855aa6943df6021db

File: src/GeniusYield/Types/Value.hs (hunk 1 above)
Comment (4TT1L4) [reply src/GeniusYield/Types/Value.hs]: Fixed: b996d9d892a6d5587920d5cdb74e29217e3264b7

**Review by sourabhxyz — APPROVED** [review]

**Comment by 4TT1L4** [comment]
> I made a mistake and inserted some tabs:
> 
> ```
> Tab character found here, and in two further locations.
> ```
> 
> I need to go through the changes and replace these with spaces.

I have addressed this problem in the following issue:
 - https://github.com/geniusyield/atlas/pull/61

The fix is available in the following PR:
 - https://github.com/geniusyield/atlas/pull/61

**Pull request merged**